src/LeanCode.ContractsGenerator/LeanCode.ContractsGenerator.csproj

```
@@ -7,6 +7,12 @@
     <RootNamespace>LeanCode.ContractsGenerator</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <PackageID Condition="'$(PackAsGlobalTool)' == 'true'">dotnet-contracts-generate</PackageID>
```

```suggestion
  <PropertyGroup Condition="'$(PackAsGlobalTool)' == 'true'">
    <PackageID>dotnet-contracts-generate</PackageID>
```
Or does it not matter that the other properties remain set?

---

It does not at all - this one overrides existing property, everything else can be set at all times.